src/index.js

```diff
@@ -0,0 +1,100 @@
+import './style.css';
+import { checkbox as checkBox, clearAll } from '../checkAndClear.js';
+import TDlist from '../todo.js';
+
+if (localStorage.getItem('list') !== null) {
+  const list = JSON.parse(localStorage.getItem('list'));
+  TDlist.displayList(list);
+}
+
+const description = document.querySelectorAll('.description');
+
+const clear = document.querySelector('.fa-sync-alt');
+clear.addEventListener('click', () => {
+  window.location.reload();
+});
+
+const getFocus = (event) => {
+  const li = event.target.parentNode;
+  const ellips = event.target.nextElementSibling;
+  const trash = ellips.nextElementSibling;
```

Comment:
:black_square_button: I would suggest creating a new function that will store the three lines of code found in both getFocus and lostFocus functions:
_const focus = (event) => {
const li = event.target.parentNode;
const ellips = event.target.nextElementSibling;
const trash = ellips.nextElementSibling;_}
Then you call the call the focus() function inside your getFocus and lostFocus functions.